### ISSUE TITLE
Can we lint a11y issues on CI?

### ISSUE THREAD

**Issue by planktonic** [bitcrowd/bitstyles#414]
Right now we have the storybook a11y-addon to check for issues while developing in the browser. It’d be great to run such checks automatically on CI too. Look into how to do that given the html is all in storybook stories

**Comment by angelikatyborska** [comment]
Note that ATM, many stories have failing a11y checks so this would also imply fixing all of those failures, which might be a lot of work.
<img width="1118" alt="Screenshot 2024-08-19 at 10 14 24" src="https://github.com/user-attachments/assets/edff5303-f204-4491-9593-dfb631d8d632">
